### ISSUE TITLE
flicker problem in iphone6 plus

### ISSUE THREAD

**Issue by carrollcai** [reactjs/react-tabs#186]
when i switched the tabs,the tabs background have a flicker in the whole tab area，the background is #333,the opacity is about 0.3.is iphone6 plus‘s trouble？

**Comment by danez** [comment]
Can you post an example to look at? Without anything to look at I cannot answer.

**Comment by carrollcai** [comment]
its no flash in andorid,iphone had.i cant shot screen,its so fast,i will try change your code or use other tab

**Comment by carrollcai** [comment]
you can try in iphone6 or iphone6plus,i guess this problem also in iphone 6s.

**Comment by carrollcai** [comment]
i guess the focus operation have an influence on tab

**Comment by carrollcai** [comment]
http://dev.miguai.top:19552/#/find/list   visit this url

**Comment by danez** [comment]
I don't have an iphone, so I cannot test. If you can figure out what is the problem, feel free to create a PR.

**Comment by carrollcai** [comment]
thank you，if i have any question，i will quiz

**Comment by brunosantanarj** [comment]
i am having the same problem, some solution ?

**Comment by JarrydTrokis** [comment]
The fix is a simple CSS change:

`-webkit-tap-highlight-color: transparent; `

**Comment by gaoqiang19514** [comment]
@JarrydTrokis thank you very much!

**Comment by waynebloss** [comment]
Any idea when this will be released?

**Comment by waynebloss** [comment]
Actually, commit 5d608aad2ddb5e00d22e48ade976586918c26f95 didn't fix this for me after I applied it manually by editing `node_modules/react-tabs/style/react-tabs.css`. I know that my manual edits got applied because I tried adding `background-color: aqua` to one of the selectors and I saw the change after re-starting my create-react-app dev server.

What finally did work for me was to add the following CSS to my local `assets/css/styles.css`:

```css
* {
  -webkit-touch-callout:none;                /* prevent callout to copy image, etc when tap to hold */
  -webkit-text-size-adjust:none;             /* prevent webkit from resizing text to fit */
  -webkit-tap-highlight-color:rgba(0,0,0,0); /* prevent tap highlight color / shadow */
  -webkit-tap-highlight-color: transparent;
  -webkit-user-select:none;                  /* prevent copy paste, to allow, change 'none' to 'text' */
}
```
My main `index.js` file now looks like this:

```jsx
import React from 'react';
import ReactDOM from 'react-dom';

import './assets/css/styles.css';
import 'react-tabs/style/react-tabs.css';

import App from './App';

ReactDOM.render(<App />, document.getElementById('root'));
```

**Comment by waynebloss** [comment]
I'm on iPhone 6s with iOS 11.4 using Safari.

**Comment by danez** [comment]
Fix now for real in 25cfa16
